tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SearchConfigSpec.scala

```diff
@@ -29,7 +31,12 @@ class SearchConfigSpec extends BaseSpec {
     "/kg/search/neuron-morphology.json",
     "/kg/search/neuron-density.json",
     "/kg/search/layer-thickness.json",
-    "/kg/search/bouton-density.json"
+    "/kg/search/bouton-density.json",
+    "/kg/search/data/simulations/simulation-campaign-configuration.json",
+    "/kg/search/data/simulations/simulation-campaign-execution.json",
+    "/kg/search/data/simulations/simulation-campaign.json",
+    "/kg/search/data/simulations/simulation.json",
+    "/kg/search/data/simulations/analysis-report-simulation.json"
```

Comment:
Some of these should be in `otherResources` in order for the first test to pass successfully.